client/ayon_blender/api/lib.py

```diff
@@ -252,19 +252,24 @@ def lsattrs(attrs: Dict) -> List:
             continue
         for node in getattr(bpy.data, coll):
             for attr, value in attrs.items():
-                avalon_prop = node.get(pipeline.AVALON_PROPERTY)
-                if not avalon_prop:
+                ayon_prop = node.get(pipeline.AVALON_PROPERTY) or (
+                    node.get(pipeline.AYON_PROPERTY)
+                )
+                if not ayon_prop:
                     continue
```

Comment:
Looking at this code - this logic is better moved to outside the `for attr, value ...` loop because it's not dependent on it at all.